### PR TITLE
Merge master - unified plan support 

### PR DESCRIPTION
**Pull request by makarandp0**
merge recent changes from master (except for one enabling unified plan) 

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: scripts/circleci-run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+case ${ENVIRONMENT} in
+dev)
+  echo "Testing against dev"
+  export ACCOUNT_SID=${ACCOUNT_SID_DEV}
+  export API_KEY_SECRET=${API_KEY_SECRET_DEV}
+  export API_KEY_SID=${API_KEY_SID_DEV}
+  export REGIONS='us1'
+  export ECS_SERVER=https://ecs.dev-us1.twilio.com
+  export WS_SERVER_INSIGHTS=wss://sdkgw.dev-us1.twilio.com/v1/VideoEvents
```
Comment (manjeshbhargav) [inline scripts/circleci-run-tests.sh]: @makarandp0 These are not necessary. The 2.x SDK will build the right urls based on the environment and region.

File: scripts/circleci-run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  ;;
+stage)
+  echo "Testing against stage"
+  export ACCOUNT_SID=${ACCOUNT_SID_STAGE}
+  export API_KEY_SECRET=${API_KEY_SECRET_STAGE}
+  export API_KEY_SID=${API_KEY_SID_STAGE}
+  export REGIONS='au1,ie1,us1'
+  export ECS_SERVER=https://ecs.stage-us1.twilio.com
+  export WS_SERVER_INSIGHTS=wss://sdkgw.stage-us1.twilio.com/v1/VideoEvents
```
Comment (manjeshbhargav) [inline scripts/circleci-run-tests.sh]: @makarandp0 These are not necessary. The 2.x SDK will build the right urls based on the environment and region.

File: lib/util/constants.js [hunk 3]
```diff
@@ -72,11 +72,6 @@ module.exports.DEFAULT_NQ_LEVEL_LOCAL = 1;
 module.exports.DEFAULT_NQ_LEVEL_REMOTE = 0;
 module.exports.MAX_NQ_LEVEL = 3;
 
-// eslint-disable-next-line no-warning-comments
-// TODO(mmalavalli): Once we decide to support Unified Plan on Chrome 72+,
-// we need to remove this constant and its references.
-module.exports.DEFAULT_CHROME_SDP_SEMANTICS = 'plan-b';
```
Comment (manjeshbhargav) [inline lib/util/constants.js]: @makarandp0 Please bring this back. Otherwise we will still be using unified plan on Chrome.

Comment (makarandp0) [reply lib/util/constants.js]: Thanks for catching this.

**Pull request merged**